index/index.go

```diff
@@ -167,6 +168,14 @@ func (n *Index) Search(pat string, opt *SearchOptions) (*SearchResponse, error)
 		}
 	}
 
+	var efre *regexp.Regexp
```

Comment:
I think our variable names are getting a tad too cryptic -- what do you think about `excludeFre`?

Reply:
Sure, I'll rename it.